processor/src/processor.py

```
@@ -379,22 +418,74 @@ def background_process():
+				has_error=True,
+				error_message=error_msg,
+			)
+
+			try:
+				create_processing_failure_issue(
+					token=token,
+					dataset_id=active_task.dataset_id,
+					stage=crashed_stage,
+					error_message=error_msg,
+				)
+			except Exception as linear_error:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)